src/index.html

```diff
@@ -8,20 +9,468 @@
     />
     <title>Catalog</title>
     <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
+      rel="stylesheet"
+      href="styles/index.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not a SCSS file. Please ensure that the correct CSS file is linked here.